### ISSUE TITLE
Demultiplexing Dorado vs Guppy

### ISSUE THREAD

**Issue by Thomieh73** [nanoporetech/dorado#630]
Hi, I am moving from using Guppy version 6.5.7 to Dorado version 0.4.2 to speed up our workflows and I see a big difference in the number reads after demultiplexing between the two tools. I wonder why that is? am I missing something?

The library that I am processing was made with the kit : SQK-RBK114-24, run on a flowcell 10.4.1. The library only contains one sample with Barcode 9. It has 1032 Pod5 files.

I am working on a HPC cluster which runs on Rocky Linux 9.1, and a slurm scheduler. 

For Guppy I run the command on a node with a P100 GPU and I use the following command to do basecalling and demultiplexing.
```
guppy_basecaller -x "cuda:all" \
        -c /cluster/projects/nn9305k/src/miniconda/envs/guppy_gpu_v6.5.7/data/dna_r10.4.1_e8.2_400bps_sup.cfg \
        --barcode_kits SQK-RBK114-24 \
		--detect_barcodes \
		--detect_mid_strand_barcodes \
		--enable_trim_barcodes \
		--num_barcoding_threads 12 \
		--num_barcoding_buffers 12 \
		--gpu_runners_per_device 24 \
        --num_callers 24 \
        --records_per_fastq 0 \
        --compress_fastq \
        --disable_pings \
        --min_qscore 7 \
        -i pod5 \
        -s fastq
```

For  Dorado I run it on a node with a A100 GPU and then I use the command:

```
dorado basecaller -x "cuda:all" --min-qscore 7 --no-trim --emit-fastq pod5/dna_r10.4.1_e8.2_400bps_sup\@v4.2.0 pod5 | dorado demux --kit-name SQK-RPB114-24 --emit-fastq --output-dir demultiplexed
```

The results of both tools

|  Tool | total reads processed | Barcode 9 reads |  unclassified reads|
| ----| ----| ----| ----| 
| Guppy |  4.127.357 |3.056.896 |1.070.010 |
| Dorado | 4.1.27.461 | 9.547 | 3.861.590 |

Other barcodes are in both cases also found, but are only assigned few reads (< 1000)

I am puzzled that dorado is throwing away so many reads? Why could that be?

I wonder if I am missing something in the workflow here.

**Comment by HalfPhoton** [comment]
Hi @Thomieh73,
It looks like you're demuxing on a different kit in dorado `guppy: SQK-RBK114-24 vs dorado: SQK-RPB114-24` 
I'm not familiar with differences in the kits but is this intentional? 

Kind regards,
Rich

**Comment by Thomieh73** [comment]
Ah wait, I did not see that. Let me try again with the correct kit.

**Comment by Thomieh73** [comment]
thanks @HalfPhoton for spotting that error. I checked and it makes sense now :-) I close this threat.
